package.json

```diff
@@ -52,7 +52,7 @@
     "sanitize:mocha:results": "node ./scripts/sanitize-mocha-results.js",
     "prestart": "yarn ensure-deps",
     "start": "cypress open --dev --global",
-    "stop-only": "npx stop-only --skip .cy,.publish,.projects,node_modules,dist,dist-test,fixtures,lib,bower_components,src,__snapshots__ --exclude cypress-tests.ts,*only.cy.js",
+    "stop-only": "npx stop-only --skip .cy,.publish,.projects,node_modules,dist,dist-test,fixtures,lib,bower_components,src,__snapshots__,patches --exclude cypress-tests.ts,*only.cy.js",
```

Comment:
Our stop-only check was failing because our patch file for mocha removes their changelog - their changelog had a 'it.only' written in it. 😝